### ISSUE TITLE
Linuxboot causes my server screen black

### ISSUE THREAD

**Issue by hanxinwu1** [linuxboot/linuxboot#76]
When I replaced the UEFI shell with the linux-4.19.6 compiled bzimage according to the example, I entered the server and then entered the uefi shell and displayed a black screen，why？

**Comment by orangecms** [comment]
What mainboard do you have and which one do you use here from the LinuxBoot repo? Many LinuxBoot implementations target serial consoles, so for a graphical one, you'd need to select more drivers.

**Comment by hanxinwu1** [comment]
> 你有什么主板，你从LinuxBoot存储库使用哪一个？许多 LinuxBoot 实现都以串行控制台为目标，因此对于图形控制台，您需要选择更多驱动程序。
I compiled u-root and Linux kernel through this website：https://github.com/linuxboot/book/tree/master/coreboot.u-root.systemboot, and then compiled the Linux kernel embedded in Intramfs into a bzimage file with a utk command to replace the uefi shell in the server bios, and then the server booted up, and after entering the ufi shell, the monitor screen was black, do you mean that my computer will be connected to the bios serial port on the server instead of the monitor to have output?

**Comment by rminnich** [comment]
Yes, our kernel config files do not usually enable graphics. But it is easy to enable graphics.

This may be enough:

CONFIG_VT=y
# CONFIG_CONSOLE_TRANSLATIONS is not set
CONFIG_VT_CONSOLE=y
CONFIG_VT_CONSOLE_SLEEP=y
CONFIG_HW_CONSOLE=y

**Comment by rminnich** [comment]
A good way to test your kernel, before putting it in flash, is to boot it as you would any other kernel, and make sure it is doing what you want.

Can you tell us what system you are doing this for?

**Comment by hanxinwu1** [comment]
> 
In fact, my kernel .config file in addition to not find CONFIG_VT_CONSOLE_SLEEP you listed the remaining config_vt are open, but still can not output any information on the monitor, my kernel menuconfig options in the Built-in kernel command line I filled in an ‘’etherealprintk=serial, ttyS0, 57600 console=ttyS0, 57600‘’, this instruction is to specify the display information to the serial port for display, so it will cause the monitor to be black screen
